backend/admin.py

```diff
@@ -127,6 +127,7 @@ def get_menu_name(self, obj):
     def get_order_name(self, obj):
         return f'Order #{obj.order.id}'
     get_order_name.short_description = 'Order'
+    get_order_name.admin_order_field = '-order_id'
```

Comment:
Should this not be just order_id? 

Locally, I get the oldest order first and if the goal is to find recent orders quickly shouldn't it be the opposite?

> First couple items
![image](https://user-images.githubusercontent.com/55285451/226444476-da77f26c-4e44-4f0f-ad6c-1b3cdcdcdb69.png)

> Last couple items
![image](https://user-images.githubusercontent.com/55285451/226444604-4053dc19-29c2-4860-b5dd-f67069b1b090.png)

Reply:
The goal was for it to be newest first indeed, and the dash was supposed to do that but I had placed it in the wrong location. The admin_order_field tells Django how it should sort the "calculated" value of get_order_name. Added an ordering-column that orders by -order_id which does what we want it to, both for order items and for orders.